setup.py

```diff
@@ -85,6 +85,8 @@
         "sqlalchemy",
         "toml",
         "uvicorn",
+        "tzdata",  # time zone stuff
+        "pyarrow",
```

Comment:
Do we really need pyarrow? We don't ever `pandas.read_parquet` in dsgrid. If you want to use it in tests, you can add it the `dev` extras in this file.